simulations/agents/liquidation_agent.py

```diff
@@ -234,29 +329,76 @@ def record(self, env):
 
 class AdversarialLiquidationAgent(LiquidationAgent):
     """
-    Liquidation agent that manipulates borrower positions via Uniswap
+    Liquidation agent that manipulates the price in Uniswap
+    to bring borrowers positions to distress
     """
 
     def __init__(
         self,
         env,
         i: int,
-        pool_implementation_abi,
-        mintable_erc20_abi,
+        pool_implementation_abi: type,
+        mintable_erc20_abi: type,
         pool_address: bytes,
         token_a_address: bytes,
         token_b_address: bytes,
-        liquidation_addresses: typing.List,
-        uniswap_pool_abi,
-        quoter_abi,
+        liquidation_addresses: List,
+        uniswap_pool_abi: type,
+        quoter_abi: type,
         swap_router_abi,
         uniswap_pool_address: bytes,
         quoter_address: bytes,
         swap_router_address: bytes,
         uniswap_fee: int,
-        aave_oracle_abi,
+        aave_oracle_abi: type,
         aave_oracle_address: bytes,
     ):
+        """
+        Initialise the Liquidator agent and create the corresponding
+        account in the EVM.
+
+        The agent stores the ABIs of the Aave contracts, the Uniswap contracts
+        and the token contracts that they will be interacting with.
+        ABIs are previously loaded using the function `:py:func:verbs.abi.load_abi`.
```

Comment:
Should be ` :py:fun:``verbs.abi.load_abi`` ` i.e the backticks should be after the `func` (should be single ticks, sorry could not work out how to show them)